tests/h/util/datetime_test.py

```diff
@@ -1,29 +1,29 @@
 import datetime
 
-from h.util.datetime import utc_iso8601, utc_us_style_date
-
-
-class Berlin(datetime.tzinfo):
-    """Berlin timezone, without DST support."""
-
-    def utcoffset(self, dt):
-        return datetime.timedelta(hours=1)
-
-    def tzname(self, dt):
-        return "Berlin"
-
-    def dst(self, dt):
-        return datetime.timedelta()
-
-
-def test_utc_iso8601():
-    t = datetime.datetime(2016, 2, 24, 18, 3, 25, 7685)
-    assert utc_iso8601(t) == "2016-02-24T18:03:25.007685+00:00"
+import pytest
 
+from h.util.datetime import utc_iso8601, utc_us_style_date
 
-def test_utc_iso8601_ignores_timezone():
-    t = datetime.datetime(2016, 2, 24, 18, 3, 25, 7685, Berlin())
-    assert utc_iso8601(t) == "2016-02-24T18:03:25.007685+00:00"
+TIMEZONE = datetime.timezone(offset=datetime.timedelta(hours=1), name="Berlin")
```

Comment:
Unsurprisingly it turns out the `datetime` library has an easier way to define a timezone than implementing a class although perhaps I'm missing why this was done, as it makes a big song and dance about not being DST?

Is this going to explode in the winter? If we're worried about out by 1 errors, should we just make this Cairo and be done with it?

Reply:
Oh nice, I have somewhere in my git stash a version of this (which was causing pylint to complain about missing methods from the abstract class) which used pytz to just pick the "Berlin" tz directly from their DB, but this is nice too :+1: